### ISSUE TITLE
iOS.csproj and Droid.csproj backslash issue in both samples

### ISSUE THREAD

**Issue by AdenFraser** [UXDivers/Grial-UI-Kit-Support#171]
Having downloaded both the full and starter samples tonight for the first time it's taken some time to get them working due to both the Droid and iOS Projects not referencing the shared project.

This turned out to be an issue in the iOS.csproj and Droid.csproj files, where backslashes had been replaced with the encoded counterparts (```%5C```):

For Instance, the following will fail to build due to 7 errors — 6 failed references to AssemblyGlobal.cs and a failed reference to the main App.cs :
```
  <ItemGroup>
    <ProjectReference Include="..%5CNewSample%5CNewSample.csproj">
      <Project>{7041522B-6406-4AA7-8C6D-8AD1661DD392}</Project>
      <Name>NewSample</Name>
    </ProjectReference>
  </ItemGroup>
```

The following, correct slashes, fixes it:
```  <ItemGroup>
    <ProjectReference Include="..\NewSample\NewSample.csproj">
      <Project>{7041522B-6406-4AA7-8C6D-8AD1661DD392}</Project>
      <Name>NewSample</Name>
    </ProjectReference>
  </ItemGroup>
```

Presumably this is a problem with the sample generator on the web side — not a problem with the actual Grial source.

**Comment by LeoHere** [comment]
@AdenFraser thanks a lot for the detailed report!

Yesterday we released the update ```2.0.52.0``` for Grial App and Nuget Packages. Now if you open the application details page in the Grial Admin site you will see a new tab that let you configure which version of Grial you want your app to be based on, as shown in the image bellow.

![image](https://cloud.githubusercontent.com/assets/16000584/25070362/49be0e6c-2272-11e7-8d53-22071ff061e2.png)
   

The issue you mentioned is fixed in ```2.0.52.0``` but still present in ```2.0.0.0```. We will fix it there ASAP.

By default, when you download the Starter/Full projects, the packages will be based on the latest Grial version available. So if you just download the app again you will get the packages without those issues.

**Comment by AdenFraser** [comment]
Cool — that version wasn't there last night when I was first getting set up (only v2.0.0.0 was available), but will keep that in mind in future.

Cheers,
